src/components/VSlider/VSlider.js

```diff
@@ -166,32 +185,47 @@ export default {
 
       const data = {
         props: {
-          color: this.validationState
+          color: this.validationState,
+          focused: !!this.validationState
         }
       }
 
       if (this.$attrs.id) data.props.for = this.$attrs.id
 
       return this.$createElement(VLabel, data, this.$slots.label || this.label)
     },
+    genListeners () {
+      return Object.assign({}, {
+        blur: this.onBlur,
+        click: this.onSliderClick,
+        focus: this.onFocus,
+        keydown: this.onKeyDown,
+        keyup: this.onKeyUp
+      }, this.$listeners)
```

Comment:
You want to combine listeners, not override them. Each of these `onXyz` methods needs to re-emit the event.